src/components/Visualization.tsx

```diff
@@ -31,15 +30,16 @@ interface IVisualization {
 const Visualization = ({ toggleCatalog }: IVisualization) => {
   // `nodes` is an array of UI-specific objects that represent
   // the Integration.Steps model visually, while `edges` connect them
+
   const defaultViewport: Viewport = {
-    x: 0,
-    y: 0,
+    // 80/2 means half of the size of the icon so the placeholder icon can be centered
+    x: window.innerWidth / 2 - 80 / 2,
+    y: (window.innerHeight - 77) / 2 - 80 / 2,
     zoom: 1.2,
   };
-
   const [isPanelExpanded, setIsPanelExpanded] = useState(false);
   const [, setReactFlowInstance] = useState(null);
-  const reactFlowWrapperRef = useRef<HTMLDivElement>(null);
+  const reactFlowWrapper = useRef(null);
```

Comment:
Why was this removed?

Reply:
this was a part of the reverting the PR: https://github.com/KaotoIO/kaoto-ui/pull/1010/files#diff-20fc6180949d2fcf17d991cc62c6be55e572335ea16c00c805dcf3c7a8a6089dL38